src/include/storage/delta_record.h

```diff
@@ -152,6 +137,54 @@ class PACKED ProjectedRow {
   }
 };
 
+/**
+ * A ProjectedRowInitializer calculates and stores information on how to initialize ProjectedRows
+ * for a specific layout.
```

Comment:
Can you describe a bit more the intended usage? I.e. the most efficient way to use InitializeProjectedRow and ProjectedRow's CopyProjectedRowLayout? is there a performance difference between the two if you have to repeatedly initialize projected rows?